google_guest_agent/agentcrypto/mtls_mds.go

```diff
@@ -34,12 +38,19 @@ const (
 	defaultCredsDir = "/etc/pki/tls/certs/mds"
 	// rootCACertFileName is the root CA cert.
 	rootCACertFileName = "root.crt"
+	// clientCertsKey is the metadata server key at which client identity certificate is exposed.
+	clientCertsKey = "instance/credentials/certs"
 )
 
 var (
 	googleRootCACertUEFIVar = uefi.VariableName{Name: googleRootCACertEFIVarName, GUID: googleGUID}
 )
 
+// MetadataServerClientInterface is the minimum required Metadata Server interface for Guest Agent MTLS bootstrapper.
+type MetadataServerClientInterface interface {
```

Comment:
Consider moving it to `metadata` package?

Reply:
done